test/unit_Polytope.jl

```diff
@@ -135,6 +135,11 @@ for N in [Float64, Rational{Int}, Float32]
     L4 = linear_map(M, P, check_invertibility=false)
     @test L4 isa VPolygon
 
+    # linear map for mixed types
+    M = [2 1; 0 1] # Int's
+    LM = linear_map(M, P)
+    @test LM isa HPolytope
```

Comment:
I would also test for the numeric type (wasn't that the whole point? :wink:)!

Reply:
The point was to make `linear_map` work; in master this gives an error. Yes, i should add a test `LM isa HPolytope{N}` because `Int` is promoted to the other `N`'s in the outer loop.